i18n/zh-CN/docusaurus-plugin-content-blog/3-streampark-usercase-bondex-paimon.md

```diff
@@ -283,10 +283,10 @@ kubectl create secret docker-registry streamparksecret
 创建 k8s secret 密钥用来拉取 ACR 中的镜像 streamparksecret 为密钥名称 自定义
 
 ```shell
-kubectl create secret docker-registry streamparksecret
---docker-server=registry-vpc.cn-zhangjiakou.aliyuncs.com
---docker-username=xxxxxx
---docker-password=xxxxxx -n streamx
+kubectl create secret docker-registry streamparksecret 
+--docker-server=registry-vpc.cn-zhangjiakou.aliyuncs.com 
+--docker-username=xxxxxx 
+--docker-password=xxxxxx -n streampark
```

Comment:
The streamx here is a command or program, unlike other simple texts, and I can't be sure if changing this will affect the actual flow of the article, so I don't recommend it

Reply:
Ok，I will resolve it.